### PR TITLE
Add an events and talks overview page

### PR DESCRIPTION
**Pull request by zner0L**

### CODE REVIEW

File: layouts/event/list.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{{ define "main" }}
+<main>
+    {{ .Content }}
+    <div class="press-releases">
```
Comment (baltpeter) [inline layouts/event/list.html:4]:
```suggestion
    <div class="events">
```

**Comment by NathanBnm** [comment]
@balpeter should this be added to French translation too?

**Comment by baltpeter** [comment]
We probably won't be having any French events or talks anytime soon but it might still be helpful to list the English ones?

**Comment by NathanBnm** [comment]
@baltpeter sure, even if it's not in France it shows that you're active for French people too :smile:

**Pull request merged**